### PR TITLE
Add onion name for CNN

### LINKED ISSUE #83: Remove onion name for iROZHLAS
This SecureDrop instance has been retired and removed from the directory.

### LINKED ISSUE #85: requesting short onion name for CNN
CNN has passed IV. They request `cnn.securedrop.tor.onion` as a short onion.
@eloquence and @conorsch

### PR DESCRIPTION
**Pull request by conorsch**
### Status

Ready for review. Contains two changes:

* Removes iROZHLAS (closes #83)
* Adds CNN (closes #85)

### Review Checklist

- [ ] Changes to `onboarded.txt` are accurate
- [ ] The file `default.rulesets.TIMESTAMP.gz` has been updated, extracting that file and inspecting the contents of the JSON file produces the expected rules
- [ ] The ruleset has been verified by modifying the HTTPS Everywhere configuration in a Tor Browser instance pointing to `Path Prefix`: `https://raw.githubusercontent.com/freedomofpress/securedrop-https-everywhere-ruleset/$BRANCH_NAME`
- [ ] `index.html` has been updated using `./update_index.sh`

### Post-Deployment Checklist

- [ ] Added/modified onion names have been updated in the SecureDrop Directory

### CODE REVIEW

File: rulesets/cnn-securedrop-ruleset.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<ruleset name="CNN">
+	<target host="cnn.securedrop.tor.onion" />
+	<rule from="^http[s]?://cnn.securedrop.tor.onion"
+    to="http://http://qmifwf762qftydprw2adbg7hs2mkunac5xrz3cb5busaflji3rja5lid.onion" />
```
Comment (eloquence) [inline rulesets/cnn-securedrop-ruleset.xml:4]: Something got borked here @conorsch , accidental protocol duplication?

**Review by eloquence — CHANGES_REQUESTED** [review]
(CNN ruleset not valid as-is due to protocol duplication)

**Comment by conorsch** [comment]
Without a doubt, not sure how that happened. Will fix and regenerate.

**Comment by conorsch** [comment]
The problem only occurs for the CNN entry because the API is returning a protocol for only that entry: 

```
$ curl -s https://securedrop.org/api/v1/directory/ | jq -r '.[].onion_address' | grep http | wc -l
1
```
and the onboarding script assumes there's no protocol on the onion field. 
See related discussion in https://github.com/freedomofpress/securedrop.org/pull/878. While we could easily update the onboarding script to be more forgiving, in this case, I'm going to edit the db entry so that the API returns consistent results across the board. At a later date, we can decide whether we want to enforce stronger validation on that field for future updates.

After updating, I'll regenerate and resign, as stated above.

**Comment by conorsch** [comment]
@eloquence Updated as described. Please test again, it should work now.

**Review by eloquence — CHANGES_REQUESTED** [review]
CNN loads for me now, but `irozhlas.securedrop.tor.onion` is still included in the ruleset XML (just removed from `onboarded.txt`), sorry :/

**Comment by conorsch** [comment]
:facepalm: I never `rm`'d the irozhlas file. Merely removing from onboarding allows the script to complete successfully, but doesn't change the contents of the tarball. Fixing...

**Comment by conorsch** [comment]
@eloquence Once more, I beseech you: test.

**Review by eloquence — APPROVED** [review]
That did the trick - thanks much :-)

**Pull request merged**